$NetBSD: patch-configure,v 1.6 2023/03/17 12:40:34 jperkin Exp $

Disable all the plugins which are installed by the nagios-plugin-* packages.

--- configure.orig	2023-01-17 22:21:31.000000000 +0000
+++ configure
@@ -15397,7 +15397,7 @@ _ACEOF
 
 fi
 
-      if test "$ac_cv_lib_radiusclient_rc_read_config" = "yes"; then
+      if false; then
         EXTRAS="$EXTRAS check_radius\$(EXEEXT)"
         RADIUSLIBS="-lradiusclient"
 
@@ -16849,7 +16849,7 @@ elif test "$ac_cv_uname_s" = "SunOS"; th
         ac_cv_ps_cols=9
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: using nagios-plugins internal ps command (pst3) for solaris" >&5
 $as_echo "using nagios-plugins internal ps command (pst3) for solaris" >&6; }
-	if test `isainfo -b` = 64 ; then
+	if test "$MACHINE_ARCH" = "x86_64"; then
 		pst3_use_64bit=1
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: using 64bit pst3" >&5
 $as_echo "$as_me: using 64bit pst3" >&6;}
@@ -18133,7 +18133,7 @@ fi
 
 
 # Check whether --with-snmpget_command was given.
-if test "${with_snmpget_command+set}" = set; then :
+if false; then :
   withval=$with_snmpget_command; PATH_TO_SNMPGET=$withval
 fi
 
@@ -18144,7 +18144,7 @@ cat >>confdefs.h <<_ACEOF
 #define PATH_TO_SNMPGET "$PATH_TO_SNMPGET"
 _ACEOF
 
-	EXTRAS="$EXTRAS check_hpjd check_snmp\$(EXEEXT)"
+	#EXTRAS="$EXTRAS check_hpjd check_snmp\$(EXEEXT)"
 else
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: Get snmpget from http://net-snmp.sourceforge.net to make check_hpjd and check_snmp plugins" >&5
 $as_echo "$as_me: WARNING: Get snmpget from http://net-snmp.sourceforge.net to make check_hpjd and check_snmp plugins" >&2;}
@@ -18205,7 +18205,7 @@ _ACEOF
 
 fi
 
-if ( $PERL -M"Net::SNMP 3.6" -e 'exit' 2>/dev/null  )
+if false
 then
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for Net::SNMP perl module" >&5
 $as_echo_n "checking for Net::SNMP perl module... " >&6; }
@@ -18368,7 +18368,7 @@ IFS=$as_save_IFS
 esac
 fi
 PATH_TO_FPING=$ac_cv_path_PATH_TO_FPING
-if test -n "$PATH_TO_FPING"; then
+if test -n "$PATH_TO_FPING6" -a "$PATH_TO_FPING6" != "no"; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: $PATH_TO_FPING" >&5
 $as_echo "$PATH_TO_FPING" >&6; }
 else
@@ -18408,7 +18408,7 @@ IFS=$as_save_IFS
 esac
 fi
 PATH_TO_FPING6=$ac_cv_path_PATH_TO_FPING6
-if test -n "$PATH_TO_FPING6"; then
+if test -n "$PATH_TO_FPING6" -a "$PATH_TO_FPING6" != "no"; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: $PATH_TO_FPING6" >&5
 $as_echo "$PATH_TO_FPING6" >&6; }
 else
@@ -18439,7 +18439,8 @@ cat >>confdefs.h <<_ACEOF
 _ACEOF
 
 	EXTRAS="$EXTRAS check_fping\$(EXEEXT)"
-	if test x"$with_ipv6" != xno && test -n "$PATH_TO_FPING6"; then
+	if test x"$with_ipv6" != xno && test -n "$PATH_TO_FPING6" -a "$PATH_TO_FPING6" != "no"; then
+
 
 cat >>confdefs.h <<_ACEOF
 #define PATH_TO_FPING6 "$PATH_TO_FPING6"
